controllers/topics.controllers.js

```diff
@@ -26,3 +27,14 @@ exports.getArticleById = (request, response, next) => {
       next(error)
     });
 };
+
+exports.getArticles = (request, response, next) => {
+  fetchArticles()
+  .then((articlesWithStringCommentCount) => {
+    const articles = articlesWithStringCommentCount.map((article) => {
+      const commentCount = Number(article.comment_count)
+      return {...article, comment_count: commentCount }
+    }) 
```

Comment:
ave a look into the CAST() command in SQL to turn this into a number. You shouldn't need this map then :)